frameworklauncher/src/test/resources/TestInputs/WrongFrameworkDescription.json

```diff
@@ -48,39 +37,45 @@
       "taskRetryPolicy": {
         "maxRetryCount": -1,
         "fancyRetryPolicy": false
-      }
-    },
-    "taskService": {
-      "version": 33,
-      "entryPoint": "HBaseRegionServer/start.bat",
-      "sourceLocations": [
-        "hdfs://NameNode2-VIP.Yarn-Prod-CO4.CO4.ap.gbl:443/HBaseRegionServer",
-        "/HBaseCommon"
-      ],
-      "resource": {
-        "cpuNumber": 28,
-        "memoryMB": 29,
-        "portRanges": [
-          {
-            "begin": 110,
-            "end": 113
-          },
-          {
-            "begin": 120,
-            "end": 123
-          }
+      },
+      "taskService": {
+        "version": 33,
+        "entryPoint": "HBaseRegionServer/start.bat",
+        "sourceLocations": [
+          "hdfs://NameNode2-VIP.Yarn-Prod-CO4.CO4.ap.gbl:443/HBaseRegionServer",
+          "/HBaseCommon"
         ],
-        "diskType": 0,
-        "diskMB": 31
+        "resource": {
+          "cpuNumber": 28,
+          "memoryMB": 29,
+          "gpuNumber": 2,
+          "gpuAttribute": 1,
+          "portDefinitions": {
+            "ht%%tp": null,
```

Comment:
>%% [](start = 15, length = 2)

what ?

Reply:
This is wrong input used to validate test

---
In reply to: [189181219](https://github.com/Microsoft/pai/pull/588#discussion_r189181219) [](ancestors = 189181219)